backend/README.md

```
@@ -72,46 +74,49 @@ JWT 인증·권한 처리, 수강신청 경쟁률·대기목록, 좌석 제한·
+
+2. [Courses (강의 관리)](#courses-강의-관리)
+    - `GET /api/admin/courses` (Read Courses)
+    - `POST /api/admin/courses` (Add Course)
+    - `GET /api/admin/courses/professor` (Get Courses By Professor)
+    - `GET /api/admin/courses/{department_id}` (Read Courses By Department)
+    - `GET /api/admin/courses/{course_id}/students` (Read Students By Course)
+    - `PUT /api/admin/courses/{course_id}` (Update Course)
+    - `DELETE /api/admin/courses/{course_id}` (Delete Course)
+    - `GET /api/admin/courses/{course_id}` (Read Course)
+
+1. [Registrations (수강 신청)](#registrations-수강-신청)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)